elastic/beats
# [Elastic Agent] Cleaner output of inspect command

## What does this PR do?

Use of classic errors instead of custom type for inspect command. custom type is more suitable for logs where it carries call stack and error history here we just need an output displayed to user.

## Why is it important?

call stack is confusing for user

## Checklist


## How to test

build agent and provide config with `management.mode=fleet` without enrolling. then run `./elastic-agent inspect`
You should see error without call stack

Fixes: #18344
